style.css

```diff
@@ -119,4 +114,842 @@
     align-content: space-between;
     padding-right: 20px;
   }
+
+  .recent-work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    width: 375px;
+    height: 3455px;
+    left: 0.01px;
+    top: 682px;
+  }
```

Comment:
- [ ] Kindly use `display: grid` instead of `display: flex;`, as the requirements asked `In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both).`